AST-496 - fix: Added transition CSS to site-identity to make shrink sticky layout smoothen (#2854)
brainstormforce/astra: changelog.txt

v3.6.0 (Unreleased)
- Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
- Improvement: Added close button for multiselect selected items in Google font variants.
- Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
- Improvement: Moved 'Submenu Animation' control from General tab to Design  tab.
- Improvement: Changed sub-menu container animation option name from 'Default' to 'None'.
- Fix: Max-width responsive CSS enlarges logo size on mobile devices in old header layout.
- Fix: Cart Icon not working with Sticky Header after enabling the option "Hide when scrolling down" on mobile devices.
- Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
- Fix: CLS – Flashing desktop menu on mobile view.
- Fix: WooCommerce - product search widget button not aligned on other than shop and single product page.

> - Fix: Sticky Header - Transition effect of headers when Shrink Effect is enable.

Reviewer: Fix: Sticky Header - Transition effect of headers when Shrink Effect is enabled.

Author: Done 👍

> - Fix: Site Identity - Improved transition effect when sticky header & shrink options enable.

Reviewer 2: ```suggestion
- Fix: Sticky Header - Transition effect of headers when Shrink Effect is enable.
```

Author: Done 👍
